app/assets/stylesheets/application.css.scss.erb

```diff
@@ -146,3 +146,11 @@ $image-path: 'img';
     display: block;
   }
 }
+
+// Search overrides based on identity-site
+
+.usa-search-input--joined-right {
+  border-top-right-radius: 0;
+  border-bottom-right-radius: 0;
```

Comment:
I hadn't noticed this before about the [search component](https://design.login.gov/components/search/), but it should ideally be fixed in the design system as well. It looks like they're intended to be squared in USWDS ([source](https://github.com/uswds/uswds/blob/0f1f4d0a12bb3981a52cc882c78b79931e6b004c/src/stylesheets/components/_search.scss#L82-L84)), but it's inadvertently overridden in our design system input styles ([source](https://github.com/18F/identity-style-guide/blob/ba2b1f7863a59a1fcbd47b713cfb7a4455813ab2/src/scss/components/_inputs.scss#L10)).

I can make a note to follow-up on this.

Ideally, we shouldn't need to have this class at all, since it should be the behavior of the default search input placed within a grid container of whatever size we prefer.

Reply:
Thanks!